src/ServiceControl.Transports.PostgreSql/QueueTableSnapshot.cs

```diff
@@ -0,0 +1,6 @@
+namespace ServiceControl.Transports.PostgreSql;
+
+public class BrokerQueueTableSnapshot(BrokerQueueTable details) : BrokerQueueTable(details.DatabaseDetails, details.QueueAddress)
+{
+    public long RowVersion { get; set; }
```

Comment:
RowVersion is still correct for Postgres?

Reply:
it's just an internal name for calculating the throughput